share/eds/cache/multicache.go

```diff
@@ -0,0 +1,51 @@
+package cache
+
+import (
+	"errors"
+
+	"github.com/filecoin-project/dagstore/shard"
+)
+
+// MultiCache represents a Cache that looks into multiple caches one by one.
```

Comment:
[suggestion]
```suggestion
// MultiCache represents a Cache that looks into two caches one by one.
```

I would also rename to something like DoubleCache as Multi implies more than two